[ARM] Compilation error

While compiling on ARM (Debian, kernel 6.1.0-21), it returns following error:

```
make[1]: Entering directory '/usr/src/linux-headers-6.1.0-21-arm64'
  CC [M]  /opt/lunatik/lib/luasocket.o
/opt/lunatik/lib/luasocket.c: In function ‘luasocket_pushaddr’:
/opt/lunatik/lib/luasocket.c:47:40: error: invalid application of ‘sizeof’ to incomplete type ‘char[]’
   47 | #define LUASOCKET_ADDRMIN(addr) (sizeof((addr)->sa_data))
      |                                        ^
/opt/lunatik/lib/luasocket.c:89:47: note: in expansion of macro ‘LUASOCKET_ADDRMIN’
   89 |                 lua_pushlstring(L, addr_data, LUASOCKET_ADDRMIN(addr));
      |                                               ^~~~~~~~~~~~~~~~~
make[2]: *** [/usr/src/linux-headers-6.1.0-21-common/scripts/Makefile.build:255: /opt/lunatik/lib/luasocket.o] Error 1
make[1]: *** [/usr/src/linux-headers-6.1.0-21-common/Makefile:2035: /opt/lunatik] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-6.1.0-21-arm64'
make: *** [Makefile:14: all] Error 2
```

--- a/lib/luasocket.c
+++ b/lib/luasocket.c
@@ -41,7 +41,7 @@ do {						\
 
 #define LUASOCKET_SOCKADDR(addr)	(struct sockaddr *)&addr, sizeof(addr)
 #define LUASOCKET_ADDRMAX		(sizeof(struct sockaddr_ll)) /* AF_PACKET */
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 2, 0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 1, 80)
 #define LUASOCKET_ADDRMIN(addr)	(sizeof((addr)->sa_data_min))
 #else
 #define LUASOCKET_ADDRMIN(addr)	(sizeof((addr)->sa_data))

